Cargo.toml

```diff
@@ -40,7 +40,7 @@ debug = 1
 debug-assertions = true # <-
 incremental = false
 lto = 'fat'
-opt-level = 'z' # <-
+opt-level = 3 # <-
```

Comment:
can we use the other size opt-level ('s') or does that also run into the rustc bug?

Reply:
That also hits the bug

Reply:
that's too bad; let's create an issue to remember to undo this when the bug is fixed